debian/snapd.postrm

```diff
@@ -41,9 +54,9 @@ if [ "$1" = "purge" ]; then
             rm -rf "/var/snap/$snap/common"
             rm -f "/var/snap/$snap/current"
             # opportunistic remove (may fail if there are still revisions left
-            for d in "/snap/bin" "/snap/$snap" "/var/snap/$snap" "/snap" "/var/snap"; do
+            for d in "/snap/bin" "/snap/$snap" "/var/snap/$snap" "/var/snap"; do
                 if [ -d "$d" ]; then
-                    rmdir --ignore-fail-on-non-empty $d
+                    rmdir --ignore-fail-on-non-empty $d || true
```

Comment:
How can it be non-empty if we remove components of `/var/snap/` above? Sorry for nitpicking but I'd rather know what happens here.